### PR TITLE
Feature/calculator

### PR DESCRIPTION
**Pull request by iruzevic**
### Added
- new constants for CPT result.

### Changed
- `getListingPageUrl` function is now mor dynamic.

### Removed
- `getFormsLocationsPageUrl` function.
- `getFormsEntriesPageUrl` function.
- `getFormsTrashPageUrl` function.

## Summary by CodeRabbit

- **New Features**
	- Introduced new constants for enhanced result output and admin page capabilities.
	- Updated URL generation logic for greater flexibility in admin page navigation.

- **Refactor**
	- Streamlined URL generation methods for improved efficiency and maintainability.

- **Chores**
	- Removed unused constants and functions, optimizing codebase clarity and performance.

### CODE REVIEW

File: src/Config/UtilsConfig.php [hunk 1]
```diff
@@ -338,6 +338,13 @@ class UtilsConfig
 	 */
 	public const SLUG_POST_TYPE = self::MAIN_PLUGIN_PROJECT_SLUG;
 
+	/**
+	 * Post type name - result output.
+	 *
+	 * @var string
+	 */
+	public const SLUG_RESULT_POST_TYPE = self::MAIN_PLUGIN_PROJECT_SLUG . '-res';
```
Comment (piqusy) [inline src/Config/UtilsConfig.php:346]:
```suggestion
	public const SLUG_RESULT_POST_TYPE = self::MAIN_PLUGIN_PROJECT_SLUG . '-result';
```
Why not like this?

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice work 👏🏼 

Left one suggestion

File: src/Helpers/UtilsGeneralHelper.php [hunk 2]
```diff
@@ -41,98 +41,69 @@ public static function getProjectVersion(): string
...
+		}
 
-		if (!empty($type)) {
-			$typeKey = "&type={$type}";
+		if (!empty($parent)) {
+			$parentKey = "&parent={$parent}";
 		}
 
-		return \get_admin_url(null, "admin.php?page={$page}{$typeKey}");
+		return \get_admin_url(null, "admin.php?page={$page}{$typeKey}{$formIdKey}{$parentKey}");
```
Comment (ghost) [inline src/Helpers/UtilsGeneralHelper.php]: Why not use something like:
```suggestion
		$args = [];
		if (!empty($parent)) {
			$args['parent'] = $parent;
		}
		return \add_query_arg($args, get_admin_url(null, 'admin.php'));
```

Comment (goranalkovic-infinum) [reply src/Helpers/UtilsGeneralHelper.php]: or just
```php
$args['parent'] = $parent;
```
😄

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Comment by dingo-d** [comment]
Just make sure you squash all those commits into one once you merge this, as they are pretty useless as is now 😕

**Review by ghost — APPROVED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

File: src/Config/UtilsConfig.php (hunk 1 above)
Comment (piqusy) [reply src/Config/UtilsConfig.php:346]: Not a valid suggestion?

**Review by piqusy — CHANGES_REQUESTED** [review]

**Review by dadadavorin — APPROVED** [review]

**Pull request merged**